src/Libraries/RevitNodes/Schedule/ScheduleField.cs

```diff
@@ -0,0 +1,52 @@
+﻿
+namespace Revit.Schedule
+{
+    /// <summary>
+    /// Revit Schedule Field
+    /// </summary>
+    public class ScheduleField
+    {
+
+        #region Internal Properties
+
+        /// <summary>
+        /// Internal reference to the Revit Element
+        /// </summary>
+        internal Autodesk.Revit.DB.ScheduleField InternalScheduleField
+        {
+            get; set;
+        }
+
+        /// <summary>
+        /// Reference to the Element
```

Comment:
You can change this to something like `Internal constructor. Used by the public static constructor to construct an instance of the ScheduleField class.`